docs/source/developers/python.rst

```diff
@@ -177,13 +186,23 @@ Using pip
    to manage your development. Please follow the conda-based development
    instructions instead.
 
-On macOS, install all dependencies through Homebrew that are required for
+On macOS, use Homebrew to install all dependencies required for
 building Arrow C++:
 
 .. code-block:: shell
 
    brew update && brew bundle --file=arrow/python/Brewfile
 
+If you find that your system is missing additional dependencies, consider
+also:
+
+.. code-block:: shell
+
+   brew bundle --file=arrow/c_glib/Brewfile
```

Comment:
Hmm... if `arrow/python/Brewfile` is missing some dependencies, shouldn't we add them there instead of telling people to install the C/GLib dependencies?

Reply:
I think a better solution would be to make a `cpp/Brewfile` to contain the C++ deps, and if GLib needs more, its Brewfile could install the cpp one first and then its others. I suspect that pyarrow doesn't require anything beyond the cpp ones so the python Brewfile would go away. 

I chose not to address that in this patch and would prefer to ticket separately, so as not to let the perfect be the enemy of the good, but happy to discuss if you disagree.

Reply:
Ok, this is done following @kou's suggestion (thank you!), and I've updated the docs accordingly.